src/BuildCheck.UnitTests/EndToEndTests.cs

```diff
@@ -242,9 +254,12 @@ public void CustomAnalyzerTest_WithEditorConfig(string analysisCandidate, string
     {
         using (var env = TestEnvironment.Create())
         {
-            var analysisCandidatePath = Path.Combine(TestAssetsRootPath, analysisCandidate);
+            string analysisCandidatePath = Path.Combine(TestAssetsRootPath, analysisCandidate);
+
+            // Can't use Transitive environment due to the need to dogfood local nuget packages.
             AddCustomDataSourceToNugetConfig(analysisCandidatePath);
-            File.WriteAllText(Path.Combine(analysisCandidatePath, EditorConfigFileName), ReadEditorConfig(
+            string editorConfigName = Path.Combine(analysisCandidatePath, EditorConfigFileName);
```

Comment:
Not for this PR - but this feels strange. Editorconfig file should be collocated with the sources - so it should be created as transient file together with the project files (then we wouldn'tt need to handle its deletion)

Reply:
I wish I could, but the problem is with local nugets that I need to use as a part of projects. These are not visible from transient folder.

Reply:
Gotcha